netutils/ntpclient/ntpclient.c

```diff
@@ -524,7 +1429,22 @@ static int ntpc_daemon(int argc, char **argv)
  ****************************************************************************/
 
 /****************************************************************************
- * Name: ntpc_start
+ * Name: ntpc_dualstack_family()
+ *
+ * Description:
+ *   Set the protocol family used (AF_INET, AF_INET6 or AF_UNSPEC)
+ *
+ ****************************************************************************/
+
+void ntpc_dualstack_family(int family)
+{
+  sem_wait(&g_ntpc_daemon.lock);
+  g_ntpc_daemon.family = family;
```

Comment:
why not use Kconfig for dualstack support? it more like a compile time decision.

Reply:
This allows e.g. cloud to tell device to limit its DNS for NTP to IPv4 or IPv6  (for debugging or any problems with server)